hack/e2e-tests.sh

```diff
@@ -3,36 +3,34 @@ SOURCE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 source $SOURCE_ROOT/hack/tests/common.sh
 
-continue=false
+parse_args() {
+  local tmp long
+  long=cacheBuild,continue,selector:,skipBuild,skipTests,test:
+  tmp=$(getopt --options x --long "$long" --name "$(basename "$0")" -- "$@") \
+    || return 1
+  eval set -- "$tmp"
+  while :; do
+    case "$1" in
+      --cacheBuild) buildOpts=; shift;;
+      --continue) continue=true; shift;;
+      --selector) NODE_SELECTOR=$2; shift 2;;
+      --skipBuild) build=false; shift;;
+      --skipTests) skipTests=true; shift;;
+      --test) test=$2; shift 2;;
+      -x) set_x=-x; set -x; shift;;
+      --) shift; break;;
+    esac
+  done
+}
```

Comment:
We also pass values from the e2e script to the tests themselves, which currently if we try do to will fail:

This should be a valid command:
`./hack/e2e-tests.sh --continue --debug --skipBuild=true --test=test.DefaultInstall`